hw-11/main.js

```diff
@@ -0,0 +1,105 @@
+// Ex1 returns an array with element elem repeated n times.
+const repeatElement = (element, repeat) => {
+    return Array(repeat).fill(element)
+  }
+  const repeatElementSecond = (element, repeat) => {
+    let newArray = []
+    for (let i = 0; i < repeat; i++) {
+      newArray = newArray.push(element)
+    }
+    return newArr
+  }
+  const repeatElementThird = (element, repeat) => {
+    let newArray = []
+    let i = 0
+    while (i < repeat) {
+      newArray.push(element)
+      i++
+    }
+    return newArray
+  }
+  
+  // Ex2 reverse Array
+  const reverseAray = (arr = []) => {
+    let newArray = []
+    for (let i = arr.length - 1; i >= 0; i--) {
+      newArray = newArray + arr[i]
+    }
+    return newArray
+  }
+  
+  // Ex3 remove Falsy (undefine, null, 0, '', false)
+  const removeFalsy = arr => arr.filter(element => element)
+  
+  // Ex4 Creat an object from an Array of key-value pairs
+  const getKeyValueSecond = (array = []) => {
+    const newObject = {}
+    array.forEach(item => {
+      newObject[item[0]] = item[1]
+    })
+    return newObject
+  }
+  
+  // Ex5 sort Array 
+  const sortArr = arr => arr.sort()
+  
+  // Ex6 check if a input is an object or not
+  const isObject = data => {
```

Comment:
```suggestion
  const checkIsObject = data => {
```

Thử chạy: `isObject(null)`